src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -87,103 +123,159 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
+                    </div>
+                    ${OFF_STATES.includes(stateObj.state)
+                      ? ""
+                      : html`
+                          <div class="playback-controls">
+                            ${!supportsFeature(stateObj, SUPPORT_PREVIOUS_TRACK)
+                              ? ""
+                              : html`
+                                  <paper-icon-button
+                                    icon="hass:skip-previous"
+                                    .action=${"media_previous_track"}
+                                    @click=${this._handleClick}
+                                  ></paper-icon-button>
+                                `}
+                            ${(stateObj.state !== "playing" &&
+                              !supportsFeature(stateObj, SUPPORTS_PLAY)) ||
+                            stateObj.state === UNAVAILABLE ||
+                            (stateObj.state === "playing" &&
+                              !supportsFeature(stateObj, SUPPORT_PAUSE) &&
+                              !supportsFeature(stateObj, SUPPORT_STOP))
+                              ? ""
+                              : html`
+                                  <paper-icon-button
+                                    class="playPauseButton"
+                                    .icon=${stateObj.state !== "playing"
+                                      ? "hass:play"
+                                      : supportsFeature(stateObj, SUPPORT_PAUSE)
+                                      ? "hass:pause"
+                                      : "hass:stop"}
+                                    .action=${"media_play_pause"}
+                                    @click=${this._handleClick}
+                                  ></paper-icon-button>
+                                `}
+                            ${!supportsFeature(stateObj, SUPPORT_NEXT_TRACK)
+                              ? ""
+                              : html`
+                                  <paper-icon-button
+                                    icon="hass:skip-next"
+                                    .action=${"media_next_track"}
+                                    @click=${this._handleClick}
+                                  ></paper-icon-button>
+                                `}
+                          </div>
+                        `}
+                  </div>
                 `}
           </div>
-          <paper-icon-button
-            icon="hass:dots-vertical"
-            @click=${this._handleMoreInfo}
-          ></paper-icon-button>
+          ${OFF_STATES.includes(stateObj.state) ||
+          !stateObj.attributes.media_duration ||
+          !stateObj.attributes.media_position ||
+          this._narrow
+            ? ""
+            : html`
+                <paper-progress
+                  .max="${stateObj.attributes.media_duration}"
+                  .value="${stateObj.attributes.media_position}"
```

Comment:
media position is static and you need to calculate it on the fly based on media state and `media_position_updated_at`. You should extract `currentProgress` method from `hass-media-player-model.ts` into `data/media_player.ts` and use that.